.env.example

```diff
@@ -0,0 +1,26 @@
+## SET ENV VARS TO BE USED IN THIS PROJECT
```

Comment:
Just for the record, the **.env** file will allow us to use this project with our without **Docker** almost seamlessly